server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -29,7 +29,7 @@ uri = postgresql://pbenchcontainer:pbench@localhost:5432/pbenchcontainer
 secret-key = "pbench-in-a-can secret shhh"
 
 [openid-connect]
-server_url = http://localhost:8090
+#server_url = http://localhost:8090
```

Comment:
While your justification is valid, depending on Nikhil's #3235 this may be short-lived. 😆 Timing may not be everything, but a lot is happening all at once ...